tests/conftest.py

```diff
@@ -308,10 +308,12 @@ def finalizer():
         Delete the PVC
         """
         for instance in instances:
+            backed_pv = instance.backed_pv
             instance.delete()
             instance.ocp.wait_for_delete(
                 instance.name
             )
+            helpers.validate_pv_delete(backed_pv)
```

Comment:
in reclaim policy 'retain' it will fail

Reply:
removed, covered as part of https://github.com/red-hat-storage/ocs-ci/pull/636/